src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.e2e.ts

```diff
@@ -71,4 +100,23 @@ describe('sbb-tooltip-trigger', () => {
 
     expect(dialog).toHaveAttribute('open');
   });
+
+  it("doesn't show tooltip on keyboard event with hover-trigger when disabled", async () => {
+    const tooltipTrigger = await page.find('sbb-tooltip-trigger >>> button');
+    const tooltip = await page.find('sbb-tooltip');
+    const dialog = await page.find('sbb-tooltip >>> dialog');
+    const changeSpy = await tooltipTrigger.spyOnEvent('focus');
+
+    element.setAttribute('disabled', true);
+    tooltip.setProperty('hoverTrigger', true);
+
+    await tooltipTrigger.focus();
```

Comment:
I'm not sure how realistic this test ist because you can't focus a disabled button (?).